library/threading.c

```diff
@@ -79,6 +82,50 @@ int (*mbedtls_mutex_unlock)( mbedtls_threading_mutex_t * ) = threading_mutex_unl
  */
 #define MUTEX_INIT  = { PTHREAD_MUTEX_INITIALIZER, 1 }
 
+#else
+
+static void threading_mutex_init_windows( mbedtls_threading_mutex_t *mutex )
+{
+    if( mutex == NULL || mutex->is_valid )
+        return;
+
+    mutex->is_valid = InitializeCriticalSectionAndSpinCount( &mutex->cs, 1024 ) != FALSE;
```

Comment:
MSDN claims `InitializeCriticalSectionAndSpinCount()` always succeeds on Vista and beyond, which is what we support in the `development` branch.

That implies the test with `FALSE` is redundant.